app.rb

```diff
@@ -0,0 +1,63 @@
+require_relative './teacher'
+require_relative './student'
+require_relative './person'
+require_relative './rental'
+require_relative './book'
+
+class App
+  def initialize
+    @books = []
+    @peoples = []
+  end
+
+  def list_books
+    @books.each { |book| puts "Title: #{book.title} Author: #{book.author} \n" }
+  end
+
+  def list_peoples
+    @peoples.each do |people|
+      puts "[#{people.class.name}] Name: #{people.name} ID: #{people.id} Age: #{people.age} \n"
+    end
+  end
+
+  def create_teacher(age, specialization, name)
+    @peoples << Teacher.new(age, specialization, name: name)
+  end
+
+  def create_student(age, name, parent_permission)
+    @peoples << Student.new(age, name: name, parent_permission: parent_permission)
+  end
+
+  def create_book
+    puts 'Title:'
+    title = gets.chomp.to_s
+    puts 'Author:'
+    author = gets.chomp.to_s
+    @books << Book.new(title, author)
+  end
+
+  def create_rental
```

Comment:
Great job @MuhammadUsmanAslam 🤝

- [ ] Great job on implementing the changes requested by the previous reviewer. Kindly ensure the rentals are created successfully and can be displayed in the UI. At the moment there is no error but the created rentals are not visible in the UI.
![rental](https://github.com/MuhammadUsmanAslam/OOP-school-library/assets/63917725/06e1c5c6-7f8f-4531-8b04-b169b61e6a4e)